geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/StringBytesGlossary.java

```diff
@@ -198,6 +192,8 @@
   public static final byte bPLUS = SIMPLE_STRING_ID; // +
   public static final byte bMINUS = ERROR_ID; // -
 
+  public static final byte[] bZERO = stringToBytes("0");
```

Comment:
There is already a `NUMBER_0_BYTE` constant defined in this class, but it's a `byte` rather than a `byte[]`. I'm not sure if there's a sensible way that it could be used instead of defining a new constant here, but it's worth a try, I think.

Reply:
`NUMBER_0_BYTE` could be replaced with `bZERO[0]` but I think that will reduce the semantics of how it reads now.